projects/packages/search/src/dashboard/components/record-meter/lib/record-info.js

```diff
@@ -65,43 +65,43 @@ export default function getRecordInfo(
 		}
 
 		// sort & split items into included and other
-		const postTypeItems = splitUsablePostTypes(
+		const PostTypeItems = splitUsablePostTypes(
```

Comment:
I think `postTypeItems` is correct here (unless PostTypeItems is a class)?

Reply:
I think so. We should use camelCase in JS.

![image](https://user-images.githubusercontent.com/1425433/171313823-d6fcc6ce-daa4-4f3f-bb63-35128b0af1c4.png)

REF: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#naming-conventions